cmd/kogito/command/project/display_project.go

```diff
@@ -64,9 +65,9 @@ func (i *displayProjectCommand) InitHook() {
 
 func (i *displayProjectCommand) Exec(cmd *cobra.Command, args []string) error {
 	log := context.GetDefaultLogger()
-	config := context.ReadConfig()
-	if config.Namespace != "" {
-		log.Infof("Using project '%s'", config.Namespace)
+	currentProject := shared.GetCurrentNamespaceFromKubeConfig()
+	if len(currentProject) > 0 {
+		log.Infof("Using project '%s'", currentProject)
```

Comment:
maybe move to the message package?

Reply:
same for the other printed messages

Reply:
I'm moving there slowly, only the new messages are being put in there. I'll review it.